DataConnectors/AWS-S3/Utils/AwsResourceCreator.ps1

```diff
@@ -54,37 +54,40 @@ function New-S3Bucket
         
         # Get s3 bucket name from user and clean up based on naming rules see https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html
         
-        $script:bucketName = (Read-ValidatedHost -Prompt "Please enter S3 bucket name" -MaxLength 64 -MinLength 3)
+        $script:bucketName = (Read-ValidatedHost -Prompt "Please enter S3 bucket name (between 3 and 63 characters long)" -MaxLength 64 -MinLength 3)
 
         Write-Log -Message "Using S3 Bucket name: $bucketname" -LogFileName $LogFileName -Indent 2
-            
+        
+        $regionConfiguration = aws configure get region
```

Comment:
Very nice! I thought this would be a good idea, but had not figured out how to do it.

Reply:
Thanks :)